home/src/main/resources/config/example.runtime.properties

```
@@ -194,3 +194,11 @@ proxy.eligibleTypeList = http://www.w3.org/2002/07/owl#Thing
 #fileUpload.maxFileSize = 10485760
 #comma separated list of mime types allowed for upload
 #fileUpload.allowedMIMETypes = image/png, application/pdf
+
+# Captcha configuration. Available implementations are: DEFAULT (text-based) and RECAPTCHA
+# If captcha.implementation property is not provided, system will fall back to DEFAULT implementation
+# For RECAPTCHA method, you have to provide siteKey and secretKey.
+# More information on siteKey and secretKey is available on: https://www.google.com/recaptcha
+captcha.implementation = DEFAULT
```

There should be captcha.enabled property with possible values true and false. The true is default one (if the property is not defined). 
Instead of DEFAULT we should use the nanocaptcha term for captcha.implementation property. Values for this property should be case insensitive (nanocaptcha and recaptcha can be also used). Also, there should be nanocaptcha.difficulty with two possible values easy and hard. Easy is the default value. 
```suggestion
captcha.implementation = nanocaptcha
nanocaptcha.difficulty = easy
```

---

I have refactored my code and implemented everything you have suggested